include/graph/Nodes.hpp

```diff
@@ -42,6 +42,7 @@ struct FormatPointsNode : Node, IPointCloudDescription
 
 	static void formatAsync(const VArray::Ptr& output, const IPointsNode::Ptr& input,
 	                        const std::vector<rgl_field_t>& fields, cudaStream_t stream);
+	static VArrayProxy<GPUFieldDesc>::Ptr getGPUFields(IPointsNode::Ptr input, const std::vector<rgl_field_t>& fields, cudaStream_t stream);
 
 	inline VArray::ConstPtr getData() const { return output; }
```

Comment:
Could be removed since we have `getFieldData()`?

Reply:
No, these two have different purposes. `getGPUFields()` is a helper function preparing instructions on how to format on the GPU. `getFieldData()` just returns the output.

Reply:
I was referring to the function `getData()` :smiley:

Reply:
Ah right. Removed ;)